bindings/Sofa/src/SofaPython3/Sofa/Types/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ SP3_add_python_module(
         DESTINATION  Sofa
         SOURCES      ${SOURCE_FILES}
         HEADERS      ${HEADER_FILES}
-        DEPENDS      Sofa.Core Sofa.DefaultType SofaPython3::Plugin SofaPython3::Bindings.Sofa.Core
+        DEPENDS      Sofa.Core SofaPython3::Plugin SofaPython3::Bindings.Sofa.Core
```

Comment:
Should be propagated to [Bindings.SofaConfig.cmake.in](https://github.com/sofa-framework/SofaPython3/blob/master/bindings/Sofa/Bindings.SofaConfig.cmake.in)